templates/zerver/dev_login.html

```diff
@@ -36,6 +36,9 @@ <h4 class="login-page-header">{{ _('Click on a user to log in!') }}</h4>
                 </div>
             </div>
         </form>
+        <div id="devtools-wrapper">
+            <a id="devtools-link" href="/devtools">Zulip Developer Tools</a>
```

Comment:
Stylistically, we usually capitalized just the first word in this sort of title.